components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -881,11 +889,15 @@ private Properties getCaptchaConfigs() {
         Properties properties = BasicAuthenticatorDataHolder.getInstance().getRecaptchaConfigs();
 
         if (properties != null && !properties.isEmpty() &&
-                Boolean.valueOf(properties.getProperty(CaptchaConstants.RE_CAPTCHA_ENABLED))) {
+                Boolean.parseBoolean(properties.getProperty(CaptchaConstants.RE_CAPTCHA_ENABLED))) {
```

Comment:
Is there a reason to edit this?

Reply:
There was a suggestion in the IntelliJ Idea on this. Will revert back to the previous one (Boolean.valueOf) if this is not recommended.
<img width="717" alt="Screenshot 2023-06-29 at 12 08 26" src="https://github.com/wso2-extensions/identity-local-auth-basicauth/assets/54329644/b90631c2-22f3-4b59-928d-f333b461906a">